client/src/components/layout/Vote.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react'
+import { withRouter } from 'react-router'
+
+const Vote = props => {
+
+  const reviewId = props.reviewId
+  const [voteCount, setVoteCount] = useState({
+    upVote: 0,
+    downVote: 0
+  })
+
+  const getVoteCount = async () => {
+    try {
+      const response = await fetch(`/api/v1/votes/${reviewId}`)    
```

Comment:
Better having this be a GET request, but even better is grabbing this data in the initial fetch for the service on it's show page! We're already also fetching the reviews for the service, we can also add in the votes for those reviews. Right now, this fetch requests is being issued for EVERY review tile needed to fetch its own vote data, which in a large app is not ideal! We can consolidate the fetch request to grab all the data at once